generator/src/main.rs

```diff
@@ -1,18 +1,18 @@
 use anyhow::Context;
-use clap::Parser;
+use clap::{Args, Parser, Subcommand};
 use concordium_rust_sdk::{
-    common::types::{Amount, TransactionTime},
+    common::types::Amount,
     endpoints::Endpoint,
-    id::types::AccountAddress,
-    types::{
-        transactions::{send, BlockItem},
-        NodeDetails, WalletAccount,
-    },
-    v2::{self, BlockIdentifier},
+    types::WalletAccount,
+    v2::{self},
 };
-use futures::TryStreamExt;
-use rand::{rngs::StdRng, Rng, SeedableRng};
-use std::{path::PathBuf, str::FromStr};
+use generator::{
+    generate_transactions, CcdGenerator, CommonArgs, MintCis2Generator,
+    RegisterCredentialsGenerator, TransferCis2Generator, WccdGenerator,
+};
+use std::{path::PathBuf, str::FromStr, sync::Arc};
+
+mod generator;
```

Comment:
It would be better to call the library part just lib. Then you don't have to name it here.

Reply:
I would prefer no to, since we would then need to move `CcdArgs`, `TransferCis2Args`, and `Mode` into lib as well, and I feel those belong in main.rs.